Services/Mail/classes/AutoResponder/AutoResponderServiceImpl.php

```diff
@@ -70,68 +88,70 @@ public function disableAutoResponder() : void
         $this->auto_responder_status = false;
     }
 
-    public function handleAutoResponderMails(int $receiver_usr_id) : void
+    public function handleAutoResponderMails(int $auto_responder_receiver_usr_id) : void
     {
-        if ($this->auto_responder_data) {
-            foreach ($this->auto_responder_data as $usr_id => $mail_options) {
-                if ($this->auto_responder_repository->exists($usr_id, $receiver_usr_id)) {
-                    $auto_responder = $this->auto_responder_repository->findBySenderIdAndReceiverId(
-                        $usr_id,
-                        $receiver_usr_id
+        if ($this->auto_responder_data === []) {
+            return;
+        }
+
+        foreach ($this->auto_responder_data as $auto_responder_sender_usr_id => $mail_options) {
+            if ($this->auto_responder_repository->exists($auto_responder_sender_usr_id, $auto_responder_receiver_usr_id)) {
+                $auto_responder = $this->auto_responder_repository->findBySenderIdAndReceiverId(
+                    $auto_responder_sender_usr_id,
+                    $auto_responder_receiver_usr_id
+                );
+            } else {
+                $auto_responder = new AutoResponder(
+                    $auto_responder_sender_usr_id,
+                    $auto_responder_receiver_usr_id,
+                    $this->normalizeDateTimezone($this->clock->now())
+                         ->sub(new DateInterval('P' . $this->global_idle_time_interval + 1 . 'D'))
```

Comment:
Wirft Fehler `non_numeric value encountered`
```suggestion
                         ->sub(new DateInterval('P' . ($this->global_idle_time_interval + 1) . 'D'))
```

Reply:
Habe es etwas anders gelöst.